src/date-picker/__integ__/date-picker-dropdown.test.ts

```diff
@@ -109,6 +109,7 @@ describe('Date picker Dropdown interactions', () => {
       await page.clickOpenCalendar();
       await page.clickDate(3, 3);
       await page.clickOpenCalendar();
+      await page.keys('\uE052'); // Corresponds to right Alt key. Workaround for an issue with useFocusVisible
```

Comment:
is this relevant for this PR?

Reply:
It is carryover from https://github.com/cloudscape-design/components/pull/31/commits

Reply:
I don't understand why this would change the keyboard navigation behavior and/or focus visibility

Reply:
I don't know either. I will remove the unrelated changes and remind the author to include those in a separate PR.